test/integration/pkg/generated/openapi_generated.go

```diff
@@ -25,42 +25,30 @@ package generated
 import (
 	spec "github.com/go-openapi/spec"
 	common "k8s.io/kube-openapi/pkg/common"
-	custom "k8s.io/kube-openapi/test/integration/testdata/custom"
 )
 
 func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
 	return map[string]common.OpenAPIDefinition{
-		"k8s.io/kube-openapi/test/integration/testdata/custom.Bac":              common.EmbedOpenAPIDefinitionIntoV2Extension(custom.Bac{}.OpenAPIV3Definition(), custom.Bac{}.OpenAPIDefinition()),
```

Comment:
Why are these gone?

Reply:
huh, no idea. Will re-run the generation and see if I can either reproduce or fix it that way.

Reply:
I reset the changes in `openapi_generated.go` as the README in test/integration suggests. However it's still strange that the generation omits some types. Taking a look now to see if anything stands out... 🤔

Reply:
Fixed (I think!) - problem was that the script that generates the API violations report did not get `testdata/custom` as input. User error 🙃